src/States.cpp

```diff
@@ -111,6 +111,14 @@ namespace ale {
 
 
   State States::getState(double time, PositionInterpolation interp) const {
+
+    // If time is in times, don't need to interpolate!
+    for (int i=0; i<m_ephemTimes.size(); i++){
```

Comment:
A linear search is going to be very slow here. This should use std::lower_bound and then compare the output instead.